packages/story-editor/src/app/media/utils/useFFmpeg.js

```diff
@@ -457,6 +457,59 @@ function useFFmpeg() {
     [getFFmpegInstance]
   );
 
+  /**
+   * Converts any audio file format to MP3 using FFmpeg.
+   *
+   * @param {File} file Original audio file object.
+   * @return {Promise<File>} Converted video file object.
+   */
+  const convertToMp3 = useCallback(
+    async (file) => {
+      //eslint-disable-next-line @wordpress/no-unused-vars-before-return -- False positive because of the finally().
+      const trackTiming = getTimeTracker('load_mp3_conversion');
+
+      let ffmpeg;
+
+      try {
+        ffmpeg = await getFFmpegInstance(file);
+
+        const tempFileName = uuidv4() + '.mp3';
+        const outputFileName = getFileName(file) + '.mp3';
+
+        await ffmpeg.run(
+          // Input filename.
+          '-i',
+          file.name,
+          // Output filename. MUST be different from input filename.
+          tempFileName
+        );
+
+        const data = ffmpeg.FS('readFile', tempFileName);
+
+        return new blobToFile(
+          new Blob([data.buffer], { type: 'audio/mpeg' }),
```

Comment:
Can we use `AUDIO_MIME_TYPE`?

Reply:
This is a `convertToMp3` function so we don't want to use `AUDIO_MIME_TYPE` here - it might change in the future to superior OGG (as soon as Safari starts fully supporting it).

Reply:
But we export/reuse them or they might change, mp3 file type will not change nor do we need to explain the meaning behind the value (like we do for numbers), here it is only for this function so using 
```js
const MP3_FILE_TYPE = 'mp3'
+
..."." + MP3_FILE_TYPE`
```
 doesn't bring any value over
```js
...".mp3"
```
in a `convertToMp3` function.
In this case it's just too much, we export `AUDIO_FILE_TYPE` because it makes sense, it might/will change/be reused.